EyeDropper/explainer.md

```diff
@@ -17,18 +17,17 @@ Even though some browsers have built-in eyedropper functionality into color inpu
-3. Provide the set of keyboard modifiers and pointer buttons pressed while selecting a pixel so that web apps can provide differentiated actions while the eyedropper tool is active, for example, selecting the background color when the SHIFT key is held and the foreground color otherwise.
-4. Allow the developer to enable and disable the eyedropper through script (subject to user activation).
-5. Keep the user in control by providing the means to exit the eyedropper mode, for example, by pressing the ESC key and ensuring the event is not cancellable by the author.
-6. Keep the user in control by requiring some explicit action, for example pressing a mouse button, to indicate which pixels will have color information returned to the web page.
-7. Allow browser implementors the freedom to implement eyedropper pixel selection UI that best fits their platform and browser.  Note that a future version of the proposal may afford web developers more control over that UI.  One example supported by Chromium-based browsers is shown below. 
+2. Allow the developer to enable the eyedropper through script (subject to user activation).
+3. Keep the user in control by providing the means to exit the eyedropper mode, for example, by pressing the ESC key and ensuring the event is not cancellable by the author.
+4. Keep the user in control by requiring some explicit action, for example pressing a mouse button, to indicate which pixels will have color information returned to the web page.
+5. Allow browser implementors the freedom to implement eyedropper pixel selection UI that best fits their platform and browser.  Note that a future version of the proposal may afford web developers more control over that UI.  One example supported by Chromium-based browsers is shown below.
 
 <img src= "preview.png" alt= "example eyedropper cursor" width="500"/>
 
 ## Non-Goals
 1. This proposal does not currently define an object model for a color, though it seems like something that would be a good addition to the web platform. 
-1. This proposal does not currently define a mechanism to allow developers to hide or show the eyedropper's pixel selection UI while remaining in eyedropoper mode, but a future version may allow that, for example, to facilitate clicking on application UI elements instead of selecting a color value.
+2. This proposal does not currently define a mechanism to allow developers to hide or show the eyedropper's pixel selection UI while remaining in eyedropoper mode, but a future version may allow that, for example, to facilitate clicking on application UI elements instead of selecting a color value.
+3. This proposal does not currently define a mechanism for providing coordinate information and keyboard modifiers pressed while selecting a pixel that the web apps could leverage to implement various functionalities.
```

Comment:
This proposal does not provide a mechanism for capturing data other than the selected pixel, such as coordinates of the selected pixel.